src/Numeric/Algebra/Complex.hs

```diff
@@ -196,7 +196,7 @@ instance (Commutative r, Rng r, InvolutiveSemiring r) => Quadrance r (Complex r)
   quadrance n = realPart $ adjoint n * n
 
 instance (Commutative r, InvolutiveSemiring r, DivisionRing r) => Division (Complex r) where
-  recip q@(Complex a b) = Complex (qq \\ a) (qq \\ b)
+  recip q@(Complex a b) = adjoint $ Complex (qq \\ a) (qq \\ b)
```

Comment:
If #26 is accepted, this can be simplified to
```suggestion
  recip z = adjoint z *. recip (quadrature z)
```
(note `r` is already supposed to be a `DivisionRing`)